[2025-04-16T14:10:55.912+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-16T14:10:56.494+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: beer_data_processing.create_dim_style manual__2025-04-16T14:04:23.930993+00:00 [queued]>
[2025-04-16T14:10:56.791+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: beer_data_processing.create_dim_style manual__2025-04-16T14:04:23.930993+00:00 [queued]>
[2025-04-16T14:10:56.793+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-04-16T14:10:57.069+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): create_dim_style> on 2025-04-16 14:04:23.930993+00:00
[2025-04-16T14:10:57.201+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=35275) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-16T14:10:57.207+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'beer_data_processing', 'create_dim_style', 'manual__2025-04-16T14:04:23.930993+00:00', '--job-id', '528', '--raw', '--subdir', 'DAGS_FOLDER/beer_data_processing_dag.py', '--cfg-path', '/tmp/tmpzzh5rmyq']
[2025-04-16T14:10:57.210+0000] {standard_task_runner.py:72} INFO - Started process 35606 to run task
[2025-04-16T14:10:57.216+0000] {standard_task_runner.py:105} INFO - Job 528: Subtask create_dim_style
[2025-04-16T14:11:07.787+0000] {job.py:229} INFO - Heartbeat recovered after 12.60 seconds
[2025-04-16T14:11:10.597+0000] {task_command.py:467} INFO - Running <TaskInstance: beer_data_processing.create_dim_style manual__2025-04-16T14:04:23.930993+00:00 [running]> on host 3e53738720c8
[2025-04-16T14:11:21.159+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='beer_data_processing' AIRFLOW_CTX_TASK_ID='create_dim_style' AIRFLOW_CTX_EXECUTION_DATE='2025-04-16T14:04:23.930993+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-16T14:04:23.930993+00:00'
[2025-04-16T14:11:21.413+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-16T14:11:21.434+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-16T14:11:21.467+0000] {logging_mixin.py:190} INFO - Current task name:create_dim_style state:running start_date:2025-04-16 14:10:56.502071+00:00
[2025-04-16T14:11:21.493+0000] {logging_mixin.py:190} INFO - Dag name:beer_data_processing and current dag run status:running
[2025-04-16T14:11:21.516+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-16T14:11:25.217+0000] {job.py:229} INFO - Heartbeat recovered after 17.80 seconds
[2025-04-16T14:11:30.177+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-04-16T14:11:35.883+0000] {logging_mixin.py:190} INFO - Створено датасет dim_style з 111 записами
[2025-04-16T14:11:36.339+0000] {python.py:240} INFO - Done. Returned value was: True
[2025-04-16T14:11:38.687+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-16T14:11:38.771+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=beer_data_processing, task_id=create_dim_style, run_id=manual__2025-04-16T14:04:23.930993+00:00, execution_date=20250416T140423, start_date=20250416T141056, end_date=20250416T141138
[2025-04-16T14:11:53.599+0000] {job.py:229} INFO - Heartbeat recovered after 21.19 seconds
[2025-04-16T14:11:58.231+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-16T14:11:58.544+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-16T14:11:58.583+0000] {logging_mixin.py:190} INFO - Dag name:beer_data_processing queued_at:2025-04-16 14:04:24.139195+00:00
[2025-04-16T14:11:58.631+0000] {logging_mixin.py:190} INFO - Task hostname:3e53738720c8 operator:PythonOperator
[2025-04-16T14:12:03.466+0000] {job.py:229} INFO - Heartbeat recovered after 15.07 seconds
[2025-04-16T14:12:14.717+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-16T14:12:17.095+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-04-16T14:12:18.148+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-16T14:12:18.520+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
